### ISSUE TITLE
Why are the batch size and number of epochs much larger than common SFT settings?

### ISSUE THREAD

**Issue by tongyx361** [MARIO-Math-Reasoning/Super_MARIO#14]
Could you explain the rationale behind? Are they results of hyper-parameter searching?

**Comment by Chen-GX** [comment]
This is our empirical setting and we did not perform a hyperparameter search of the batch size. You can try a smaller batch size, but this usually requires a smaller learning rate.

**Comment by Chen-GX** [comment]
The number of epochs is also our empirical setting. The larger the number of epochs, the smoother the decrease of learning rate. Maybe you can try a smaller number of epochs.

**Comment by tongyx361** [comment]
Thanks for your explanation!
